packages/dnb-eufemia/src/extensions/forms/Form/SubmitIndicator/SubmitIndicatorDocs.ts

```diff
@@ -6,6 +6,16 @@ export const SubmitIndicatorProperties: PropertiesTableProps = {
     type: 'string',
     status: 'required',
   },
+  showLabel: {
+    doc: 'If `true`, the label will be shown.',
```

Comment:
```suggestion
    doc: 'If `true`, the label will be shown. Defaults to `false`',
```

Not sure how important it is to add, but thought it's nice to say that it's by default false.